backend/data/processors/manualFixes/ECONFixes.py

```diff
@@ -0,0 +1,192 @@
+    CONDITIONS["ECON4202"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4205"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4208"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4301"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4302"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4303"][PROCESSED] = ECON_4303()
+    CONDITIONS["ECON4307"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4309"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4310"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4311"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+    CONDITIONS["ECON4350"][PROCESSED] = ECON_4100_3_6_4202_5_8_4301_2_7_9_11_50()
+
+    # Updates the files with the modified dictionaries
+    dataHelpers.write_data(
+        CONDITIONS, "data/finalData/conditionsProcessed.json")
+    dataHelpers.write_data(COURSES, "data/finalData/coursesProcessed.json")
+
+# TODO: implement your functions here
+# NOT SURE HOW THIS WORKS (FOR THE EXCEPT PART)
+def ECON_1101(code):
+    """
+    "original": "Excluded: Students in BCom single or double-degree programs, except for 3155 Actl/Comm and 3521 Comm/Econ, are not allowed to enrol in this course. <br/><br/>"
+    
+    "processed": ""
+    """
+
+    COURSES[code]["exclusions"]["COMM#"] = 1
+    
+    return ""
+
+# NOT SURE HOW THIS WORKS (FOR THE EXCEPT PART)
+def ECON_1203(code):
+    """
+    "original": "Excluded: MATH2841, MATH2801, MATH2901, MATH2099, ACTL2002 & ACTL2131. <br/>Also, must not be enrolled in a BCom single or double degree (except 3155 Actl/Comm or 3521 Comm/Econ), or program 3715 or 3764<br/><br/>"
+    
+    "processed": "Must not be Program 3715 || 3764"
+    """
+    
+    COURSES[code]["exclusions"]["MATH2841"] = 1
+    COURSES[code]["exclusions"]["MATH2801"] = 1
+    COURSES[code]["exclusions"]["MATH2901"] = 1
+    COURSES[code]["exclusions"]["MATH2099"] = 1
+    COURSES[code]["exclusions"]["ACTL2002"] = 1
+    COURSES[code]["exclusions"]["ACTL2131"] = 1
+    COURSES[code]["exclusions"]["COMM#"] = 1
+    COURSES[code]["exclusions"]["3715"] = 1
+    COURSES[code]["exclusions"]["3764"] = 1
+
+    return ""
+
+def ECON_2206():
+    """
+    "original": "Pre-requisite conditions: ECON1203 or COMM1190 or ECON2403 or MATH1041 or MATH1231 or MATH1241 or MATH1251<br/><br/>"
+
+    "processed": "conditions: ECON1203 || COMM1190 || ECON2403 || MATH1041 || MATH1231 || MATH1241 || MATH1251"
+    """
+
+    return "ECON1203 || COMM1190 || ECON2403 || MATH1041 || MATH1231 || MATH1241 || MATH1251"
+
+def ECON_2403():
+    """
+    "original": "Prerequisites: PPEC1001 AND ECON1202 AND in program 3478 or 4797<br/><br/>"
+    "processed": "PPEC1001 && ECON1202 && in program 3478 || 4797"
+    """
+
+    return "PPEC1001 && ECON1202 && 3478 || 4797"
```

Comment:
Remember to surround in brackets, (3478 || 4797)! Or else the logic is ambiguous